ui/components/component-library/modal-header/modal-header.tsx

```diff
@@ -23,6 +24,10 @@ export const ModalHeader: React.FC<ModalHeaderProps> = ({
   return (
     <HeaderBase
       className={classnames('mm-modal-header', className)}
+      padding={4}
+      childrenWrapperProps={{
+        width: BlockSize.Full,
+      }}
```

Comment:
This fixes the issue where if there is no `startAccessory` or `endAccessory` in the `ModalHeader`'s case no back or close button the text will be centered still